.github/workflows/bot.yml

```diff
@@ -2,9 +2,11 @@ name: Benchmark
 
 on:
   push:
+    paths-ignore: ["docs/**", "**.md"]
```

Comment:
Won't this just not run the benchmarks if any docs file is changed?

Reply:
Will it run if a md file and an rs file is changed?

Reply:
> Won't this just not run the benchmarks if any docs file is changed?

This **_will_** run if any file other than `docs/**` or `**.md` is changed. If only `docs/**` or `**.md` files are changed, it will not trigger the workflow.

> Will it run if a md file and an rs file is changed?

Yes, as a `rs` file would be outside the `md`/`docs` target.

According to their [official example](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-excluding-paths):
> When all the path names match patterns in `paths-ignore`, the workflow will not run. If any path names do not match patterns in `paths-ignore`, even if some path names match the patterns, the workflow will run.